lib/public/views/lhcPeriods/Overview/LhcPeriodsOverviewModel.js

```diff
@@ -108,5 +138,47 @@ export class LhcPeriodsOverviewModel extends Observable {
     reset() {
         this._lhcPeriods = RemoteData.NotAsked();
         this._pagination.reset();
+        this._namesFilterModel.reset();
+        this.fetchLhcPeriods();
+    }
+
+    /**
+     * Register a new filter model
+     * @param {FilterModel} filterModel the filter model to register
+     * @return {void}
+     * @private
+     */
+    _registerFilter(filterModel) {
+        filterModel.visualChange$.bubbleTo(this);
+        filterModel.observe(() => {
+            if (filterModel.isValid) {
+                this.fetchLhcPeriods();
+            }
+        });
+    }
+
+    /**
+     * Toggle filter visibility
+     * @return {void}
+     */
+    toggleFiltersVisibility() {
+        this._filtersVisible = !this._filtersVisible;
+        this.notify();
+    }
+
+    /**
+     * States wheter filters panel should be displayed
+     * @return {boolean} true if filters panel should be displayed
+     */
+    get areFiltersVisible() {
+        return this._filtersVisible;
+    }
+
+    /**
+     * States whether any filter is active
+     * @return {boolean} true if any filter is active
+     */
+    isAnyFilterActive() {
+        return !this._namesFilterModel.isEmpty();
     }
```

Comment:
Probably not necessary anymore?

The last one we do not use is until we have proper info to display into it.